README.md

```diff
@@ -129,102 +121,126 @@ service:
-    type: MovieId
+    status-code: 404 type: MovieId
 ```
 
-### Generating an SDK
+## Generating an SDK
 
-To generate SDKs, you can log in with GitHub from the CLI:
+All SDKs are generated server side, so you don't have to worry about installing all the prerequisite tools locally. Before you can start generating files, you'll need to log in:
 
-```bash
+```
 fern login
 ```
 
-You can add [generators](compiler/generators) using `fern add`. By default, this
-will publish your SDK to the Fern npm registry (npm.buildwithfern.com).
+You'll be prompted to log in in a new browser tab using your GitHub account.
 
-```bash
-fern add fern-typescript-sdk
+After you have logged in, simply run the generate command to generate your SDKs:
+
+```
+fern generate
 ```
 
-To generate the TypeScript SDK, run:
+After your code is generated, the resulting artifacts are copied to the output directory specified in `generators.yml` , which we will touch on in the next section.
+
+It is possible to generate the code on your own machine by using the `--local` flag. This does require you to run Docker on your local machine, and Fern will schedule containers to do the code generation automatically. This is further specified in the CLI reference below.
+
+## Adding Generators
+
+You can add ﻿generators using the `fern add`. You can have multiple generators for a single project, e.g. a single `fern generate` call could generate SDKs in Python, TypeScript and Java all in one go.
+
+For instance, if you'd like to add a TypeScript generator to your project, simply run:
 
-```bash
-fern generate
+```
+fern add fern-typescript-sdk
 ```
 
-And voila! You just built and published a TypeScript SDK.
+Which will add the Node variant of the TypeScript SDK to the list of generators - the other variant being a TypeScript browser library, to be used by frontends directly.
 
-**Next step:** define _your_ API in Fern. Check out our [docs](https://www.buildwithfern.com/docs/definition) to learn more.
+Your `generators.yml` file should contain the new generator:
 
-# CLI reference
+```
+default-group: local
+groups:
+  local:
+    generators:
+      - name: fernapi/fern-typescript-node-sdk
+        version: 0.6.4
+        output:
+          location: local-file-system
+          path: ../../generated/typescript
+```
 
-### `fern generate [--group <group>] [--version <version>]`
+Note the `local-file-system` output may not be present by default. If it's not, simply add it and specify a local path to where the generated client will be saved.
 
-`fern generate` runs the compiler. It will validate your API and run the
-generators specified in `generators.yml`.
+## Publishing Clients
 
-If you don't have a `default-group` specified in `generators.yml`, then you must
-specify a group using the `--group` option.
+Fern supports automatically publishing clients to an NPM registry. By default, Node packages are published to `npm.buildwithfern.com` and are named `@organisation-fern/package`
```

Comment:
Want to include this line from the website?

`Whether you call them SDKs, client libraries, bindings, or wrappers, Fern generates prewritten code that developers can use to‍ build client applications more easily.`

Reply:
I've included it in the capabilities section instead, felt like the right place for it.